internal/testdata/dlasqtest/commands.txt

```diff
@@ -0,0 +1,9 @@
+# Advice: nDiv in dlasq2 can be an effective counter for isolating a bug by using os.Exit(1) and "STOP" when nDiv hits a certain number.
+
+gfortran -o dlasq1test dlamch.f lsame.f xerbla.f ieeeck.f iparmq.f ilaenv.f dlasrt.f dlaisnan.f disnan.f dlasq6.f dlasq5.f dlasq4.f dlasq3.f dlasq2.f dcopy.f dlas2.f dlascl.f dlasq1.f testdlasq1.f90
```

Comment:
Change this into a `Makefile` with a `clean` target so that the repository can be easily reverted into a clean state, or is it not worth the hassle?